### DISCUSSION TITLE
Robocopy?

### DISCUSSION THREAD

**Discussion by nosar77** [moudey/Shell#427] [General]
Has anyone figured out a way to replace copy with just robo copy on shell? This would be an awesome feature to use robo copy as default and not regular copy.

**Comment by RubicBG** [comment]
if you make sample commands for Robocopy in PowerShell that you find useful, I'll convert them to NS script
https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy

**Reply by pa-0** [reply]
I use the following to imitate Mac's "New folder with selection"  Would love to be able to set it up in Shell instead:

```bat
@echo off
cd /D "%~dp1"
md "%~n1"
for %%f in (%*) do move %%f "%~dp1%~n1"
```

**Reply by RubicBG** [reply]
```js
item(title='New folder with selection' type='file|dir' mode='multiple' cmd-line='/c set "newFolder=New folder with selection" && call md "%newFolder%" && for %a in (@sel) do call move "%a" "%newFolder%"' window='hidden')
item(title='New folder "@sel.title" with selected files' type='file|dir' mode='multiple' cmd-line='/c set "newFolder=@sel.title" && call md "%newFolder%" && for %a in (@sel) do call move "%a" "%newFolder%"' window='hidden')

**Reply by pa-0** [reply]
THANK YOU
